mantidproject/mantid
# move Elwin and Iqt tabs into Indirect Data Manipulation

**Description of work.**

This PR moves the Elwin and Iqt tabs into the indirect data manipulation interface

**To test:**

1. open indirect Data Manipulation
2. there should now be an Elwin tab and an Iqt tab.
3. go to the Elwin Tab
4. Load a reduced or sqw file,
5. chagne range selectors and check that the integration range updates
6. click run and check outputs are created.
7. go to iqt tab
8. load a sample and resolution
9. change rangle markers and check ranges in the propperty table update
10. click run and check output runs correctly.

Fixes #34628
Fixes #34629 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
